docs/standard/datetime/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Dates, times, and time zones
+description: Dates, times, and time zones
+keywords: .NET, .NET Core
+author: stevehoag
+manager: wpickett
+ms.date: 07/22/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 76e6cacc-1c0c-4a71-8cb8-018c112385ba
+---
+
+# Dates, times, and time zones
```

Comment:
I think this is bigger issue with everything on MSDN - we'll need to set a priority for the rework.